dora/core/common/src/main/java/alluxio/membership/AlluxioEtcdClient.java

```diff
@@ -109,6 +102,10 @@ public AlluxioEtcdClient(AlluxioConfiguration conf) {
     List<String> endpointsList = conf.getList(PropertyKey.ETCD_ENDPOINTS);
```

Comment:
If the `AlluxioEtcdClient` is meant to be used as a singleton then its constructor should not be public, right?

Reply:
Yeah it was used in testcontainers UT before, let me add the @VisibleForTesting annotation.